fabtests/scripts/runfabtests.py

```diff
@@ -294,6 +297,7 @@ def main():
                         help="only run tests which match the given substring expression.")
     parser.add_argument("--html", type=str, help="path to generated html report")
     parser.add_argument("--junit-xml", type=str, help="path to generated junit xml report")
+    parser.add_argument("--junit-logging", type=str, help="Write captured log messages to JUnit report: one of no|log|system-out|system-err|out-err|all")
```

Comment:
Thank you!

The function `add_argument` has an argument `choices`, which allow you to list possible values for this argument. It seems perfect to use it here.

https://docs.python.org/3/library/argparse.html#choices

Reply:
Done, thanks!